src/Collection.php

```diff
@@ -215,28 +222,30 @@ public function dropAllIndexes()
      * @param array $condition query condition
      * @param array $fields fields to be selected
      * @param array $options query options (available since 2.1).
+     * @param array $execOptions -> goto Command::executeQuery()
```

Comment:
Need a better description about what could be passed in this parameter. Alternatively, {@see Command::executeQuery()} could be used.